src/ripple/app/ledger/impl/OpenLedger.cpp

```diff
@@ -140,13 +143,22 @@ OpenLedger::apply_one (Application& app, OpenView& view,
 {
     if (retry)
         flags = flags | tapRETRY;
-    if ((router.getFlags(
-        tx->getTransactionID()) & SF_SIGGOOD) ==
-            SF_SIGGOOD)
-        flags = flags | tapNO_CHECK_SIGN;
```

Comment:
I'm leaving this instance out because when the Transactor calls checkValidity, the same flags will get checked, so this instance is redundant.